src/input/classes.scss

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import './index';
+
+.iui-input {
+  @include iui-input;
+}
+
+.iui-input-with-icon.iui-input-with-icon {
```

Comment:
This was the fix to the ordering issue in `classes.scss` discussed [here](https://github.com/iTwin/iTwinUI/pull/620#discussion_r862136220).

Reply:
:(